schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/SchemaRegistryResource.java

```diff
@@ -101,9 +102,22 @@
 
     // reserved as schema related paths use these strings
     private static final String[] reservedNames = {"aggregate", "versions", "compatibility"};
+    private final SchemaRegistryVersion schemaRegistryVersion;
 
-    public SchemaRegistryResource(ISchemaRegistry schemaRegistry, AtomicReference<LeadershipParticipant> leadershipParticipant) {
+    public SchemaRegistryResource(ISchemaRegistry schemaRegistry,
+                                  AtomicReference<LeadershipParticipant> leadershipParticipant,
+                                  SchemaRegistryVersion schemaRegistryVersion) {
         super(schemaRegistry, leadershipParticipant);
+        this.schemaRegistryVersion = schemaRegistryVersion;
+    }
+
+    @GET
+    @Path("/version")
+    @ApiOperation(value = "Get version info of this Schema Registry instance",
+            response = SchemaRegistryVersion.class)
+    @Timed
```

Comment:
nit: Can we add a tag here, so that this API would show under appropriate category in swagger?